packages/lab/src/list-next/ListNext.tsx

```diff
@@ -0,0 +1,187 @@
+import { makePrefixer, useForkRef, useId } from "@salt-ds/core";
+import {
+  Children,
+  cloneElement,
+  FocusEventHandler,
+  forwardRef,
+  HTMLAttributes,
+  isValidElement,
+  KeyboardEventHandler,
+  MouseEvent,
+  MouseEventHandler,
+  ReactElement,
+  useMemo,
+} from "react";
+import { clsx } from "clsx";
+import {
+  ListItemNext,
+  ListItemNext as DefaultListItem,
+} from "./ListItemNext";
+import { useList } from "./useList";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import listNextCss from "./ListNext.css";
+
+const withBaseName = makePrefixer("saltListNext");
+const defaultEmptyMessage = "No data to display";
+
+export interface ListNextProps extends HTMLAttributes<HTMLUListElement> {
+  /**
+   * If true, all items in list will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Use to override the default empty message.
+   */
+  emptyMessage?: string;
+  /**
+   * The component used to render a ListItem instead of the default. This must itself render a ListItem,
+   * must implement props that extend ListItemProps and must forward ListItem props to the ListItem.
+   */
+  ListItem?: ReactElement;
+  /**
+   * If `true`, the component will have no border.
+   */
+  borderless?: boolean;
+  /**
+   * If `true`, items in list will be deselectable.
+   */
+  deselectable?: boolean;
+  /**
+   * Optional id prop
+   *
+   * Used as suffix of List id: `list-{id}`
```

Comment:
I assume the user would want to provide their own id so they could select it with `getElementById` or some similar use-case. Changing the user provided id seems odd.

Reply:
Just checked and you are not using it like the comment suggests :D

Reply:
Updated the docs